engine/src/main/java/org/terasology/rendering/opengl/AbstractFBOsManager.java

```diff
@@ -234,6 +234,25 @@ public FBO get(ResourceUrn fboName) {
     }
 
     /**
+     * Returns an FBOConfig given its name.
+     *
+     * If no FBOConfig maps to the given name, null is returned and an error is logged.
+     *
+     * @param fboName a ResourceUrn representing the name of an FBO
+     * @return an FBOConfig instance if one is found associated with the given fboName, null otherwise
+     */
+    @Override
+    public FBOConfig getFboConfig(ResourceUrn fboName) {
+        FBOConfig fboConfig = fboConfigs.get(fboName);
+
+        if (fboConfig == null) {
+            logger.error("Failed to retrieve FBOConfig '" + fboName + "'!");
```

Comment:
Hmmm. I'm thinking we might want to turn this into a logger.warning instead. I'd use the error only if the program cannot proceed, while in this case the consumer code is responsible for reacting to a null return: this particular piece of code can cope perfectly well with it.